DOCS-7513-SSO-domain-redirects (#93)
mulesoft/docs-access-management: modules/ROOT/pages/managing-users.adoc

If your identity provider supports uploading service-provider metadata, you can click the *Anypoint service provider metadata* link to download your SAML configuration settings in an XML file. You can then upload this file to your identity provider to configure Anypoint Platform. Note that you must have a SAML 2.0 identity configuration to export this data.

> == SSO Redirects using Cookies

Reviewer: ```suggestion
== SSO Redirects Using Cookies
```

Reviewer: I'm assuming that "Redirects" is being used here as a noun and not a verb?

That said, I wonder if it would be more precise to add something to the title that tells what about this thing is contained in the narrative: SSO Redirect Process Using Cookies? How SSO Uses Cookies to Return You to Where You Started (no, not that...but you see where I am going with this...)?

Author: ```suggestion
== SSO Redirect Process Using Cookies
```

> Anypoint Platform uses cookies to ensure SSO users are redirected to the same location from which they initiated their login.

Reviewer: ```suggestion
Anypoint Platform uses cookies to ensure that SSO users are redirected to the same location from which they initiated their login.
```
See "ensure, ensure that" in style ref.

Reviewer: Do you think it would be helpful to introduce the following sentences as a numbered sequence of actions (ordered list) that occur to ensure that users are eventually left where they started? Because this text seems to focus on PROCESS.
It could even be illustrated by a numbered graphic, but I suspect it is not worth the effort.

Reviewer: Know that this is just an idea for you to consider, not better than what you have here, which is perfectly fine.

When a user logs in to Anypoint Platform, a cookie is created for the login session: `mulesoft.sess=xxxxx`.
The cookie contains the location, such as a public portal, from which the user initiated the login.
The cookie is passed to the user's browser.
The user is directed to their SSO provider for authentication.

> After the user is authenticated using SSO, the provider directs the user back to Anypoint Platform with the `receive-id` SAML Response.

Author: ```suggestion
After the user is authenticated using SSO, the provider directs the user back to Anypoint Platform with the authorization decision SAML response.
```

> The session cookie, `mulesoft.sess=xxxxx`, is passed from the browser, which re-directs the user back to the location from which they initiated the login process.

Reviewer: ```suggestion
The session cookie, `mulesoft.sess=xxxxx`, is passed from the browser, which redirects the user back to the location from which they initiated the login process.
```

> After the user successfully logs in to Anypoint Platform, the `mulesoft.sess` cookie is reset to default based on the OWASP standard.

Reviewer: ```suggestion
After the user successfully logs in to Anypoint Platform, the `mulesoft.sess` cookie is reset to its default, based on the OWASP standard.
```
OR
```suggestion
After the user successfully logs in to Anypoint Platform, the `mulesoft.sess` cookie returns to its default value, based on the OWASP standard.
```
